packages/now-client/src/utils/index.ts

```diff
@@ -140,10 +135,10 @@ export const fetch = async (
 
   if (opts.teamId) {
     const parsedUrl = parseUrl(url, true);
```

Comment:
You could replace `parseUrl(url, true)` with `new URL(url)` I believe.

Reply:
Good point